src/packages/client/src/runtime/getPrismaClient.ts

```diff
@@ -122,7 +125,6 @@ export interface PrismaClientOptions {
   __internal?: {
     debug?: boolean
     hooks?: Hooks
-    useUds?: boolean
```

Comment:
Why removing useUds? Some users need it, that would break for them

Reply:
Note, I see it's defined in EngineConfig

Reply:
:point_down: